src/BugsnagUnity/SimpleJson.cs

```diff
@@ -1636,7 +1636,7 @@ private static bool CanAdd(MemberInfo info, out string jsonKey)
 
 #endif
 
-    namespace Reflection
```

Comment:
Does this actually need to be changed, since its in a nested namespace?

Reply:
This was not required - I've kept the namespace as is.